Robust.Client/Robust.Client.csproj

```
@@ -24,6 +23,19 @@
     <PackageReference Include="Robust.Natives" Version="0.1.1" />
     <PackageReference Include="TerraFX.Interop.Windows" Version="10.0.20348-rc2" />
   </ItemGroup>
+  <Choose>
+    <When Condition="'$(UseSystemSqlite)' == 'True'">
+      <ItemGroup>
+        <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="6.0.8" />
+        <PackageReference Include="SQLitePCLRaw.provider.sqlite3" Version="2.1.1" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <PackageReference Include="Microsoft.Data.Sqlite" Version="6.0.2" />
+      </ItemGroup>
+    </Otherwise>
+  </Choose>
```

Simpler way to do it.

```suggestion
    <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="6.0.9" />
    <PackageReference Include="SQLitePCLRaw.provider.sqlite3" Version="2.1.2" Condition="'$(UseSystemSqlite)' == 'True'" />
    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.1.2" Condition="'$(UseSystemSqlite)' != 'True'" />
  </ItemGroup>
```

---

I've force-pushed with the suggested change; this works fine on FreeBSD.

For non-system SQLite platforms, this changes Microsoft.Data.Sqlite 6.0.2 -> Microsoft.Data.Sqlite.Core 6.0.8 and adds QLitePCLRaw.bundle_e_sqlite3 2.1.2. Would you mind checking that this doesn't do anything surprising on the other platforms?